README.md

```diff
@@ -6,8 +6,8 @@ This mod helps modders to bring attention to players about the StopModReposts ca
 ---
 # Forge Version
 
-## What does 'StopModReposts' do?
-It adds this beautiful GUI, in order to educate more users about the problem of mod reposting sites.
+## What does this Mod do?
+StopModReposts adds this beautiful GUI, in order to educate more users about the problem of mod reposting sites.
```

Comment:
Please change this to: „**The StopModReposts Splash-Screen Mod** adds this beautiful GUI (…)“

Reply:
Done, what about the title `What does this Mod do?`, is that fine?

Reply:
That‘s fine! I‘m just taking a quick look over the rest and then I can approve it! Thanks so much 🙏

Reply:
Always happy to help 😁